environment.yaml

```diff
@@ -14,3 +14,4 @@ dependencies:
 - zarr
 - pip:
   - git+https://github.com/dgergel/xsd@pointwisedownscaler_interimfix
+  - xclim
```

Comment:
Could we change this to install via `conda` rather than `pip`?

(Concretely, I mean removing the indent and moving this line to just above the `- zarr` line.)